packages/babylon/src/plugins/flow.js

```diff
@@ -2272,4 +2274,101 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         /* isAsync */ true,
       );
     }
+
+    readToken_mult_modulo(code: number): void {
+      // '%*'
+      let type = code === charCodes.asterisk ? tt.star : tt.modulo;
+      let width = 1;
+      let next = this.input.charCodeAt(this.state.pos + 1);
+      const exprAllowed = this.state.exprAllowed;
+
+      // Exponentiation operator **
+      if (code === charCodes.asterisk && next === charCodes.asterisk) {
+        width++;
+        next = this.input.charCodeAt(this.state.pos + 2);
+        type = tt.exponent;
+      }
+
+      if (next === charCodes.equalsTo && !exprAllowed) {
+        width++;
+        type = tt.assign;
+      }
+
+      if (
+        code === charCodes.asterisk &&
+        next === charCodes.slash &&
+        this.state.hasFlowComment
+      ) {
+        this.state.hasFlowComment = false;
+        this.state.pos += 2;
+        this.nextToken();
+        return;
+      }
+
+      this.finishOp(type, width);
+    }
+
+    skipBlockComment(): void {
+      if (
+        this.plugins.flow &&
```

Comment:
Can you use `this.hasPlugin("flow")` (or `"flowComments"`), like it is done in the other parts of Babylon?